src/Middlewares/RoleMiddleware.php

```diff
@@ -3,21 +3,25 @@
 namespace Spatie\Permission\Middlewares;
 
 use Closure;
-use Illuminate\Support\Facades\Auth;
 
 class RoleMiddleware
 {
     public function handle($request, Closure $next, $role)
     {
-        if (Auth::guest()) {
+        if (auth()->guest()) {
```

Comment:
I like the use of the facade better in this situation. Please revert back to the facade everywhere you replace it by the function.